publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/persistence/AnalyticsIntegrationHandler.java

```diff
@@ -0,0 +1,117 @@
+package no.unit.nva.publication.events.handlers.persistence;
+
+import static no.unit.nva.publication.events.handlers.expandresources.ExpandDataEntriesHandler.EXPANDED_ENTRY_UPDATED_EVENT_TOPIC;
+import static nva.commons.core.attempt.Try.attempt;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.net.URI;
+import java.util.Optional;
+import no.unit.nva.events.handlers.DestinationsEventBridgeEventHandler;
+import no.unit.nva.events.models.AwsEventBridgeDetail;
+import no.unit.nva.events.models.AwsEventBridgeEvent;
+import no.unit.nva.events.models.EventReference;
+import no.unit.nva.expansion.model.ExpandedResource;
+import no.unit.nva.s3.S3Driver;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.JsonUtils;
+import nva.commons.core.paths.UnixPath;
+import nva.commons.core.paths.UriWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import software.amazon.awssdk.services.s3.S3Client;
+
+public class AnalyticsIntegrationHandler extends DestinationsEventBridgeEventHandler<EventReference, EventReference> {
+
+    public static final String EXPECTED_EVENT_TOPIC_ERROR_MESSAGE =
+        "The event topic is not the expected. Expected topic is "
+        + EXPANDED_ENTRY_UPDATED_EVENT_TOPIC;
+    public static final String ANALYTICS_ENTRY_PERSISTED_EVENT_TOPIC = "PublicationService.ExpandedEntry.Analytics";
+    public static final String CONTEXT = "@context";
+    public static final EventReference EMPTY_EVENT = null;
+    public static final String TYPE_FIELD = "type";
+    private final static Logger logger = LoggerFactory.getLogger(AnalyticsIntegrationHandler.class);
+    private final S3Client s3Client;
+
+    @JacocoGenerated
+    public AnalyticsIntegrationHandler() {
+        this(S3Driver.defaultS3Client().build());
+    }
+
+    public AnalyticsIntegrationHandler(S3Client s3Client) {
+        super(EventReference.class);
+        this.s3Client = s3Client;
+    }
+
+    @Override
+    protected EventReference processInputPayload(EventReference input,
+                                                 AwsEventBridgeEvent<AwsEventBridgeDetail<EventReference>> event,
+                                                 Context context) {
+        if (topicIsInvalid(input)) {
+            logErrorMessageAndThrowException(event);
+        }
+        var s3Driver = createS3Driver(input);
+        var inputFileLocation = new UriWrapper(input.getUri()).toS3bucketPath();
+        return readPublicationAndRemoveJsonLdContext(inputFileLocation, s3Driver)
```

Comment:
Wrap this in createEventReference() or something to reduce noise in process method